pkgs/tools/security/paros/default.nix

```
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchzip, jdk8, jre8, ant, makeWrapper }:
```

Please override these in `all-packages.nix`.
```suggestion
{ lib, stdenv, fetchzip, jdk, ant, makeWrapper }:
```